uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -248,6 +248,7 @@ function tasksassignments_civicrm_permission(&$permissions) {
   $permissions += array(
     'delete Tasks and Documents' => $prefix . ts('delete Tasks and Documents'),
     'access Tasks and Assignments' => $prefix . ts('access Tasks and Assignments'),
+    'access Tasks and Assignments Files' => $prefix . ts('access Tasks and Assignments Files'),
```

Comment:
Just a question, I added a new permission in civihr for another ticket and added it to a default permissions file. Is this different here?